File: SCRIPTS/1.6/classe_pessoa_get_set.py
class Pessoa:
    def __init__(self, nome, idade, cpf):
        self._nome = nome
        self._idade = idade
        self._cpf = cpf

    @property
    def nome(self):
        return self._nome

    @property
    def idade(self):
        return self._idade

    @property
    def cpf(self):
        return self._cpf

    @nome.setter
    def nome(self, nome):
        self._nome = nome

    @idade.setter
    def idade(self, idade):
        self._idade = idade

    @cpf.setter
    def cpf(self, cpf):
        self._cpf = cpf

    def __str__(self):
        return f'Nome: {self._nome}\nIdade: {self._idade}\nCpf: {self._cpf}'


pessoa = Pessoa('Ingride', 45, 999999999)
print(pessoa)
print('-----------------')

pessoa.nome = 'Manu'
pessoa.idade = 78
pessoa.cpf = 4243465646
print(pessoa)

File: SCRIPTS/1.6/classe_pessoa_get_set.js
class Pessoa {
    constructor(nome, idade, cpf){
        this._nome = nome;
        this._idade = idade;
        this._cpf = cpf;
    }

    get nome() {
        return this._nome;
    }

     get idade() {
        return this._idade;
    }

    get cpf() {
        return this._cpf;
    }

    set nome(nome) {
        this._nome = nome
    }

    set idade(idade) {
        this._idade = idade
    }

    set cpf(cpf) {
        this._cpf = cpf
    }
}

let pessoa = new Pessoa('Ingride', 36, 90909009)
console.log(pessoa)
console.log('-----------------')

pessoa.nome = 'Luca'
pessoa.idade = 89
pessoa.cpf = 89899889
console.log(pessoa)